04-onion-routing.md

```diff
@@ -262,6 +262,9 @@ This is a more flexible format, which avoids the redundant `short_channel_id` fi
     2. data:
         * [`32*byte`:`payment_secret`]
         * [`tu64`:`total_msat`]
+    1. type: 10 (`pub_key`)
```

Comment:
What about `outgoing_node_id`?
I already have added this field in the [trampoline proposal](https://github.com/lightningnetwork/lightning-rfc/pull/654) (I believe also with TLV type 10) so I'm interested in getting it spec-ed anyway :)

Reply:
I don't think that it is a good idea to have two unapproved prs reference each other. Suppose this PR would be approved first, then renaming the field is easy enough and won't change the wire protocol.